app/helpers/api/integrations_helper.rb

```diff
@@ -230,7 +230,8 @@ def proxy_rules_preview(owner, path: nil)
     proxy_rules = owner.proxy_rules
     last_rule = proxy_rules.last
     return 'None' unless last_rule
-    code = content_tag(:code) { "#{path}#{last_rule.pattern} => #{last_rule.metric.name}" }
+    preview = "#{path unless path == '/'}#{last_rule.pattern} => #{last_rule.metric.name}"
```

Comment:
I think you can just use 
```suggestion
    preview = "#{File.join(path.to_s, last_rule.pattern.to_s)} => #{last_rule.metric.name}"
```